docs/api/1.x/openapi.rst

```diff
@@ -0,0 +1,62 @@
+.. _openapi_spec:
+
+OpenAPI Specification
+#####################
+
+The `OpenAPI Specification <https://github.com/OAI/OpenAPI-Specification>`_
+(formally known as Swagger specification)
+is a standard to describe REST APIs in a human and computer readable format
+and allow several features like interactive documentation and automated
+client generation.
+
+.. important::
+    OpenAPI 2.0 currently doensn't some features that are present on Kinto API.
+    Some known limitations are
+    lack of validation on or clauses (e.g. provide `data` or `permissions` in PATCH
+    operations), no support for non schema querystrings, as used on :ref:`filtering <filtering>`
```

Comment:
`, and provide request or default in /batch requests`

Also, I'd like to see a comment about whatever we ended up doing with additionalProperties and what bugs that fixes in what libraries, etc.